ext/MitsubaLoader/CMitsubaLoader.h

```diff
@@ -238,6 +256,8 @@ class CMitsubaLoader : public asset::IAssetLoader
 
 	protected:
 		asset::IAssetManager* manager;
+		//TODO need one packer per format class
+		core::smart_refctd_ptr<ICPUTexturePacker> m_texPacker;
```

Comment:
goes into the SContext!

Reply:
actually best if it goes into the global metadata that will be returned after the assets load.